### DISCUSSION TITLE
Custom domain first load speed is 7 seconds, vercel domain, 0.5 seconds, why?

### DISCUSSION THREAD

**Discussion by George-field** [vercel/community#3891] [Help] [answered]
### Summary

Hey All, 

I have a site deployed on Vercel using NextJS. On the domain the vercel provides, the site's load speed is rapid, we are talking .05 seconds or less. However, on a custom domain from Namecheap, it takes 7 seconds to load. Why is this and what can I check/inspect to get to the bottom of the issue?

Both Domains point to the same code.

Custom Domain: [linkable.so](https://linkable.so/)
Vercel Domain: https://linkable-git-main-georgeaffield.vercel.app/

### Steps to Reproduce

Just head over to the website, check the load speeds on both, one is much faster than the other.

**Comment by amyegan** [comment, marked as answer]
Hey @George-field. The site loaded almost instantly for me at the custom domain. The same as the vercel.app subdomain version. Do you still see a difference in loading times between the two?

**Reply by George-field** [reply]
You are right, it's a caching issue on my local machine, it has been a long day hahaha!

**Comment by George-field** [comment]
No issue, it was a local caching issue on my laptop
